opencanary_correlator/common/notifications.py

```diff
@@ -46,3 +47,13 @@ def notify(incident):
         for to in sms_numbers:
             logger.debug('SMS sent to %s' % to)
             sms.send(to, incident.format_report_short())
+
+    if c.config.getVal('console.slack_notification_enable', default=False):
+        logger.debug('Slack notifications enabled')
+        webhooks = c.config.getVal('console.slack_notification_webhook', default=[])
+        for to in webhooks:
+            response = requests.post(
+                to, json={"text": incident.format_report_short()}
```

Comment:
Just a thought here (I am going to accept the PR). We could do some formatting of the slack webhook for slack. If you check canarytokens, you will see how we do it there.

Reply:
Sure, I will have a look.